lms/djangoapps/certificates/models.py

```diff
@@ -745,10 +761,20 @@ class CertificateTemplateAsset(TimeStampedModel):
     )
     asset = models.FileField(
         max_length=255,
-        upload_to='certificate_template_assets',
+        upload_to=template_assets_path,
         help_text=_(u'Asset file. It could be an image or css file.'),
     )
 
+    def save(self, *args, **kwargs):
```

Comment:
@asadiqbal08 I'm not able to understand purpose of overriding `save` method. Could you please explain?